static/front-style.css

```
@@ -18,8 +18,60 @@
   background-color: #4caf50;
   color: white;
   cursor: pointer;
+/* Responsiveness for desktop */
+/* Containers */
+.smaily-layout-1-container, .smaily-layout-2-container {
```

WordPress coding standards require you to write every rule on a separate line:

```suggestion
.smaily-layout-1-container,
.smaily-layout-2-container {
```